From 3ca55ffdd7eb28682a65a76b72069ee7ce350377 Mon Sep 17 00:00:00 2001
From: Joey <joey@lineageos.org>
Date: Tue, 15 May 2018 22:20:35 +0200
Subject: [PATCH 14/15] SystemUI: update heads up, sync and volume panel
 drawables

Use rounded icons (from material.io) to match the other SysUI assets style

Change-Id: I54f88f0d757f631b818992513458544843e183b6
Signed-off-by: Joey <joey@lineageos.org>
---
 .../res/drawable/ic_qs_heads_up_off.xml       | 14 +++++++++++---
 .../res/drawable/ic_qs_heads_up_on.xml        | 17 ++++++++++++++---
 .../SystemUI/res/drawable/ic_qs_sync_off.xml  | 19 +++++++++++--------
 .../SystemUI/res/drawable/ic_qs_sync_on.xml   | 10 +++++-----
 .../res/drawable/ic_qs_volume_panel.xml       |  6 +++---
 5 files changed, 44 insertions(+), 22 deletions(-)

diff --git a/packages/SystemUI/res/drawable/ic_qs_heads_up_off.xml b/packages/SystemUI/res/drawable/ic_qs_heads_up_off.xml
index 904ccdfb70a..10bb4061fc8 100644
--- a/packages/SystemUI/res/drawable/ic_qs_heads_up_off.xml
+++ b/packages/SystemUI/res/drawable/ic_qs_heads_up_off.xml
@@ -19,7 +19,15 @@ Copyright (C) 2014 The Android Open Source Project
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
 
-    <path
-        android:fillColor="#FFFFFFFF"
-        android:pathData="M11.5,22.0c1.1,0.0 2.0,-0.9 2.0,-2.0l-4.0,0.0C9.5,21.1 10.4,22.0 11.5,22.0zM18.0,10.5c0.0,-3.1 -2.1,-5.6 -5.0,-6.3L13.0,3.5C13.0,2.7 12.3,2.0 11.5,2.0C10.7,2.0 10.0,2.7 10.0,3.5l0.0,0.7C9.5,4.3 9.0,4.5 8.6,4.7l9.4,9.4L18.0,10.5zM17.7,19.0l2.0,2.0l1.3,-1.3L4.3,3.0L3.0,4.3l2.9,2.9C5.3,8.2 5.0,9.3 5.0,10.5L5.0,16.0l-2.0,2.0l0.0,1.0L17.7,19.0z" />
+        <path
+            android:pathData="M0,0h24v24H0V0z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M12,22c1.1,0,2-0.9,2-2h-4C10,21.1,10.89,22,12,22z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M18,11c0-3.07-1.64-5.64-4.5-6.32V4c0-0.83-0.67-1.5-1.5-1.5S10.5,3.17,10.5,4v0.68c-0.24,0.06-0.47,0.15-0.69,0.23 L18,13.1V11z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M5.41,3.35L4,4.76l2.81,2.81C6.29,8.57,6,9.73,6,11v5l-1.29,1.29C4.08,17.92,4.52,19,5.41,19h12.83l1.74,1.74l1.41-1.41 L5.41,3.35z" />
 </vector>
diff --git a/packages/SystemUI/res/drawable/ic_qs_heads_up_on.xml b/packages/SystemUI/res/drawable/ic_qs_heads_up_on.xml
index 9c5983d4d6e..b107602787c 100644
--- a/packages/SystemUI/res/drawable/ic_qs_heads_up_on.xml
+++ b/packages/SystemUI/res/drawable/ic_qs_heads_up_on.xml
@@ -19,7 +19,18 @@ Copyright (C) 2014 The Android Open Source Project
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
 
-    <path
-        android:fillColor="#FFFFFFFF"
-        android:pathData="M11.5,22.0c1.1,0.0 2.0,-0.9 2.0,-2.0l-4.0,0.0C9.5,21.1 10.4,22.0 11.5,22.0zM18.0,16.0l0.0,-5.5c0.0,-3.1 -2.1,-5.6 -5.0,-6.3L13.0,3.5C13.0,2.7 12.3,2.0 11.5,2.0C10.7,2.0 10.0,2.7 10.0,3.5l0.0,0.7c-2.9,0.7 -5.0,3.2 -5.0,6.3L5.0,16.0l-2.0,2.0l0.0,1.0l17.0,0.0l0.0,-1.0L18.0,16.0z"/>
+        <path
+            android:pathData="M0,0h24v24H0V0z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M18,16v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-0.83-0.68-1.5-1.51-1.5S10.5,3.17,10.5,4v0.68C7.63,5.36,6,7.92,6,11v5 l-1.3,1.29C4.07,17.92,4.51,19,5.4,19h13.17c0.89,0,1.34-1.08,0.71-1.71L18,16z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M11.99,22c1.1,0,2-0.9,2-2h-4C9.99,21.1,10.88,22,11.99,22z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M6.77,4.73C7.19,4.35,7.2,3.7,6.8,3.3l0,0c-0.38-0.38-1-0.39-1.39-0.02C3.7,4.84,2.52,6.96,2.14,9.34 C2.05,9.95,2.52,10.5,3.14,10.5h0c0.48,0,0.9-0.35,0.98-0.83C4.42,7.73,5.38,6,6.77,4.73z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M18.6,3.28c-0.4-0.37-1.02-0.36-1.4,0.02l0,0c-0.4,0.4-0.38,1.04,0.03,1.42c1.38,1.27,2.35,3,2.65,4.94 c0.07,0.48,0.49,0.83,0.98,0.83c0.61,0,1.09-0.55,0.99-1.16C21.47,6.96,20.3,4.85,18.6,3.28z" />
 </vector>
diff --git a/packages/SystemUI/res/drawable/ic_qs_sync_off.xml b/packages/SystemUI/res/drawable/ic_qs_sync_off.xml
index ccb5b1db2fa..e34f67b2f02 100644
--- a/packages/SystemUI/res/drawable/ic_qs_sync_off.xml
+++ b/packages/SystemUI/res/drawable/ic_qs_sync_off.xml
@@ -21,12 +21,15 @@
         android:viewportWidth="24"
         android:viewportHeight="24">
 
-    <path
-        android:fillColor="#FFFFFFFF"
-        android:pathData="M10 6.35V4.26c-.8 .21 -1.55 .54 -2.23 .96 l1.46 1.46c.25-.12 .5 -.24 .77
--.33zm-7.14-.94l2.36 2.36C4.45 8.99 4 10.44 4 12c0 2.21 .91 4.2 2.36 5.64L4
-20h6v-6l-2.24 2.24C6.68 15.15 6 13.66 6 12c0-1 .25-1.94 .68 -2.77l8.08 8.08c-.25
-.13 -.5 .25 -.77 .34 v2.09c.8-.21 1.55-.54 2.23-.96l2.36 2.36 1.27-1.27L4.14
-4.14 2.86 5.41zM20 4h-6v6l2.24-2.24C17.32 8.85 18 10.34 18 12c0 1-.25 1.94-.68
-2.77l1.46 1.46C19.55 15.01 20 13.56 20 12c0-2.21-.91-4.2-2.36-5.64L20 4z" />
+        <path
+            android:pathData="M0,0h24v24H0V0z M0,0h24v24H0V0z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M10,5.74V5.55c0-0.68-0.71-1.11-1.32-0.82c-0.19,0.09-0.36,0.2-0.54,0.3l1.46,1.46C9.84,6.31,10,6.04,10,5.74z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M20,12c0-2.21-0.91-4.2-2.36-5.64l1.51-1.51C19.46,4.54,19.24,4,18.79,4H14v4.79c0,0.45,0.54,0.67,0.85,0.35l1.39-1.39 C17.32,8.85,18,10.34,18,12c0,0.85-0.18,1.66-0.5,2.39l1.48,1.48C19.62,14.72,20,13.41,20,12z" />
+        <path
+            android:fillColor="#FFFFFFFF"
+            android:pathData="M3.57,4.7L3.57,4.7c-0.39,0.39-0.39,1.02,0,1.41l1.65,1.65C4.45,9,4,10.44,4,12c0,2.21,0.91,4.2,2.36,5.64l-1.51,1.51 C4.54,19.46,4.76,20,5.21,20H9.5c0.28,0,0.5-0.22,0.5-0.5v-4.29c0-0.45-0.54-0.67-0.85-0.35l-1.39,1.39C6.68,15.15,6,13.66,6,12 c0-1,0.26-1.93,0.69-2.76l8.07,8.07c-0.01,0.02-0.01,0.02-0.01,0.04C14.32,17.47,14,17.83,14,18.26v0.18 c0,0.68,0.71,1.11,1.32,0.82c0.31-0.14,0.61-0.31,0.9-0.49l1.87,1.87c0.39,0.39,1.02,0.39,1.41,0l0,0c0.39-0.39,0.39-1.02,0-1.41 L4.98,4.7C4.59,4.31,3.96,4.31,3.57,4.7z" />
 </vector>
diff --git a/packages/SystemUI/res/drawable/ic_qs_sync_on.xml b/packages/SystemUI/res/drawable/ic_qs_sync_on.xml
index 05113f01ceb..a207b8a1b6d 100644
--- a/packages/SystemUI/res/drawable/ic_qs_sync_on.xml
+++ b/packages/SystemUI/res/drawable/ic_qs_sync_on.xml
@@ -20,9 +20,9 @@
         android:viewportWidth="24"
         android:viewportHeight="24">
 
-    <path
-        android:fillColor="#FFFFFFFF"
-        android:pathData="M12 4V1L8 5l4 4V6c3.31 0 6 2.69 6 6 0 1.01-.25 1.97-.7 2.8l1.46 1.46C19.54 15.03
-20 13.57 20 12c0-4.42-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6 0-1.01 .25 -1.97 .7
--2.8L5.24 7.74C4.46 8.97 4 10.43 4 12c0 4.42 3.58 8 8 8v3l4-4-4-4v3z" />
+        <path
+            android:pathData="M0.01,0h24v24h-24V0z" />
+        <path
+            android:fillColor="#FFFFFF"
+            android:pathData="M12,4V2.21c0-0.45-0.54-0.67-0.85-0.35L8.35,4.65c-0.2,0.2-0.2,0.51,0,0.71l2.79,2.79C11.46,8.46,12,8.24,12,7.79V6 c3.31,0,6,2.69,6,6c0,0.79-0.15,1.56-0.44,2.25c-0.15,0.36-0.04,0.77,0.23,1.04v0c0.51,0.51,1.37,0.33,1.64-0.34 C19.8,14.04,20,13.04,20,12C20,7.58,16.42,4,12,4z M12,18c-3.31,0-6-2.69-6-6c0-0.79,0.15-1.56,0.44-2.25 c0.15-0.36,0.04-0.77-0.23-1.04l0,0C5.7,8.2,4.84,8.38,4.57,9.05C4.2,9.96,4,10.96,4,12c0,4.42,3.58,8,8,8v1.79 c0,0.45,0.54,0.67,0.85,0.35l2.79-2.79c0.2-0.2,0.2-0.51,0-0.71l-2.79-2.79C12.54,15.54,12,15.76,12,16.21V18z" />
 </vector>
diff --git a/packages/SystemUI/res/drawable/ic_qs_volume_panel.xml b/packages/SystemUI/res/drawable/ic_qs_volume_panel.xml
index cdea5c66e1f..e7c51d2a61c 100644
--- a/packages/SystemUI/res/drawable/ic_qs_volume_panel.xml
+++ b/packages/SystemUI/res/drawable/ic_qs_volume_panel.xml
@@ -18,10 +18,10 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
         android:width="64dp"
         android:height="64dp"
-        android:viewportWidth="48"
-        android:viewportHeight="48">
+        android:viewportWidth="24"
+        android:viewportHeight="24">
 
     <path
         android:fillColor="#FFFFFFFF"
-        android:pathData="M6.0,18.0l0.0,12.0l8.0,0.0l10.0,10.0L24.0,8.0L14.0,18.0L6.0,18.0zm27.0,6.0c0.0,-3.53 -2.04,-6.58 -5.0,-8.05l0.0,16.11c2.96,-1.48 5.0,-4.53 5.0,-8.06zM28.0,6.46l0.0,4.13c5.78,1.72 10.0,7.07 10.0,13.41s-4.22,11.69 -10.0,13.41l0.0,4.13c8.01,-1.82 14.0,-8.97 14.0,-17.54S36.01,8.28 28.0,6.46z"/> <!--TODO Needs own icon-->
+        android:pathData="M3,10v4c0,0.55,0.45,1,1,1h3l3.29,3.29c0.63,0.63,1.71,0.18,1.71-0.71V6.41c0-0.89-1.08-1.34-1.71-0.71L7,9H4 C3.45,9,3,9.45,3,10z M16.5,12c0-1.77-1.02-3.29-2.5-4.03v8.05C15.48,15.29,16.5,13.77,16.5,12z M14,4.45v0.2 c0,0.38,0.25,0.71,0.6,0.85C17.18,6.53,19,9.06,19,12s-1.82,5.47-4.4,6.5c-0.36,0.14-0.6,0.47-0.6,0.85v0.2 c0,0.63,0.63,1.07,1.21,0.85C18.6,19.11,21,15.84,21,12s-2.4-7.11-5.79-8.4C14.63,3.37,14,3.82,14,4.45z"/>
 </vector>
-- 
2.17.0

